### DISCUSSION TITLE
Simple image gallery

### DISCUSSION THREAD

**Discussion by simgunz** [razonyang/hugo-theme-bootstrap#239] [Q&A] [answered]
Hi, I am looking for a way to show a simple image gallery and I was wondering if the theme already supports this feature. In the features of the theme there is `Image Viewer` but I could not find any documentation or examples on how to use it. Is it a sort of image gallery? How is it used?

**Comment by razonyang** [comment, marked as answer]
It doesn't support  image gallery so far, I'm going to use a third-party library for implementing this feature, any suggestions are welcome.

**Reply by simgunz** [reply]
[GLightBox](https://biati-digital.github.io/glightbox/) seems interesting and it has MIT license. 
~~Just out of curiosity what is the `Image Viewer` mentioned in the features?~~ found out

**Reply by razonyang** [reply]
I finally pick the Viewer.js to implement it.
I created PR #240, please feel free to test it and share your feedback.

Documentation: https://github.com/razonyang/hugo-theme-bootstrap/tree/feature-image-viewer#image-viewer
Preview(by clicking any unlinkable images): https://deploy-preview-240--hugo-theme-bootstrap.netlify.app
